eclipse-theia/theia
# monaco: fix quick-input-list styling

#### What it does


Fixes: https://github.com/eclipse-theia/theia/issues/10922#issue-1179539726.

The pull-request removes superfluous styling from #10544 which caused issues in monaco after the latest upgrade:

1. fixes the peek widget styling

![image](https://user-images.githubusercontent.com/40359487/159933566-066e9a09-5203-49a4-a9f2-7e8a07ef4180.png)

2. fixes the styling of codicons as tail icons in the quick-input
 
https://user-images.githubusercontent.com/40359487/159934055-a4b172aa-f6b1-430c-82b0-ea9b87dd8c27.mp4

#### How to test


1. confirm that the `peek` widget styling is correct - (ex: `peek definition`)
3. confirm that codicons in the quick-input menu work well (ex: `run task...`)

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
